src/GitVersionExe.Tests/GitVersionHelper.cs

```diff
@@ -2,26 +2,33 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Text;
+
```

Comment:
Hi,

Have you already read my new article? I think that's what you need at the moment, please, read it here <http://describe.jamiecooks.com/e2lndncz>

p.kiers@home.nl